### DISCUSSION TITLE
local token replace script

### DISCUSSION THREAD

**Discussion by milkshakeuk** [qetza/replacetokens-task#44] [Show and tell]
Hi, I thought I would share a bash script I wrote in case anyone might find it useful,  it replaces tokens locally but retrieves the variables from azure DevOps variable groups, which although crude mimics what this ADO pipeline task does but allows you to run locally which in certain circumstances is very helpful.

https://gist.github.com/milkshakeuk/81fbf48ad5a2414391093a868d7a7bba#comments

Please let me know your thoughts.

**Comment by qetza** [comment]
Hi @milkshakeuk,
Seams good, I'll just make sure that the `TEMP_VARIABLES_FILE` file is always deleted even if some other command fails as the file could have secrets.
